packages/components/src/components/quickfact/quickfact.stories.ts

```diff
@@ -0,0 +1,405 @@
+                      : `<div slot='${slot}' class="${cx(
+                          'slot slot--border slot--background h-6',
+                          slot === 'summary' ? 'w-[100%]' : 'w-6'
+                        )}"></div>`,
+                  title: slot
+                }
+              ]
+            }
+          },
+          constants: [
+            { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
+            { type: 'attribute', name: 'open', value: slot === 'collapse-icon' || slot === 'default' ? true : false },
+            { type: 'attribute', name: 'expandable', value: 'true' }
+          ],
+          args: overrideArgs({ type: 'slot', name: 'default', value: '' }, args)
+        })
+      )}
+    `;
+  }
+};
+
+export const Parts = {
+  parameters: {
+    controls: { exclude: ['base', 'header', 'summary', 'summary-icon', 'summary-border', 'content', 'content__slot'] }
+  },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'template',
+          name: 'sd-quickfact::part(...){outline: solid 2px red}',
+          values: ['base', 'header', 'summary', 'summary-icon', 'summary-border', 'content', 'content__slot'].map(
+            part => {
+              const outlineOffset = part === 'summary-border' ? '' : 'outline-offset: -2px';
+              return {
+                title: part,
+                // Added an outline-offset to make the outline visible for content__slot
+                value: `<style>#part-${part} sd-quickfact::part(${part}){outline: solid 2px red; ${outlineOffset};}</style><div id="part-${part}">%TEMPLATE%</div>`
+              };
+            }
+          )
+        }
+      },
+      constants: [
+        { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
+        { type: 'attribute', name: 'open', value: true },
+        { type: 'attribute', name: 'expandable', value: 'true' }
+      ],
+      args
+    });
+  }
+};
+
+/**
+ * sd-quickfacts are fully accessibile via keyboard.
+ */
+
+export const Mouseless = {
+  render: (args: any) => {
+    return html`<div class="mouseless">
+      ${generateTemplate({ args, constants: { type: 'attribute', name: 'expandable', value: 'true' } })}
+    </div>`;
+  },
+
+  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
+    const el = canvasElement.querySelector('.mouseless sd-quickfact');
+    await waitUntil(() => el?.shadowRoot?.querySelector('header'));
+    await userEvent.type(el!.shadowRoot!.querySelector('header')!, '{space}', { pointerEventsCheck: 0 });
+  }
+};
+
+/**
+ * This is a sample of how to group sd-quickfacts in a grid layout. Additional JavaScript is used to enable closing all other quickfacts when one is opened and to equalize the height of all summaries in a row. Open the "Show code" section to see the detailed implementation.
+ */
+export const Sample = {
```

Comment:
![CleanShot 2024-08-02 at 16 10 08@2x](https://github.com/user-attachments/assets/e6b32cac-7a0c-450d-8f7f-25b4bab301b7)

I think this should always appear in the row, shouldn't it?